tendrl/commons/objects/node/flows/stop_services/__init__.py

```diff
@@ -23,50 +24,64 @@ def run(self):
             )
             srv = NS.tendrl.objects.Service(service=service)
             if not srv.running:
-                logger.log(
-                    "debug",
-                    NS.publisher_id,
-                    {
-                        "message": "%s not running on "
-                        "%s" % (service, NS.node_context.node_id)
-                    },
-                    job_id=self.parameters['job_id'],
-                    flow_id=self.parameters['flow_id'],
-                )
+                if len(srv.error) > 0:
+                    raise FlowExecutionFailedError(
+                        "Unable to check status of service %s "
+                        "on %s. Error: %s" % (
+                            service,
+                            NS.node_context.node_id,
+                            srv.error
+                        )
+                    )
+                else:
+                    logger.log(
+                        "debug",
+                        NS.publisher_id,
+                        {
+                            "message": "%s not running on "
+                            "%s" % (service, NS.node_context.node_id)
+                        },
+                        job_id=self.parameters['job_id'],
+                        flow_id=self.parameters['flow_id'],
+                    )
                 continue
 
             _cmd_str = "systemctl stop %s" % service
             cmd = cmd_utils.Command(_cmd_str)
-            err, out, rc = cmd.run()
+            out, err, rc = cmd.run()
             if err:
```

Comment:
https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/cmd_utils.py#L81 cmd utils return out, err rc. While disabling service it always raises error related removed "removed symlink", That is why we previously gave like that. But that is not the correct approach, so now i have modified log level to debug to avoid this in log file.